resources/views/settings/audit.blade.php

```diff
@@ -41,12 +41,19 @@
                     </div>
                 @endforeach
 
-                <div class="form-group ml-auto"
+                <div class="form-group ml-auto mr-m"
                      component="submit-on-change"
                      option:submit-on-change:filter='[name="user"]'>
                     <label for="owner">{{ trans('settings.audit_table_user') }}</label>
                     @include('form.user-select', ['user' => $listDetails['user'] ? \BookStack\Auth\User::query()->find($listDetails['user']) : null, 'name' => 'user', 'compact' =>  true])
                 </div>
+
+
+                <div class="form-group ml-auto">
+                    <label for="ip">{{ trans('settings.ip_address') }}</label>
+                    @include('form.text', ['name' => 'ip', 'model' => $listDetails['ip'] ? \BookStack\Actions\Activity::query()->where(['ip' => $listDetails['ip']])->first() : null])
+                    <input type="submit" style="display: none">
+                </div>
```

Comment:
Currently there's a bug in the input field:

- If you perform an IP-filtered search which has no results, the input value will disappear. 

This is likely due to this line:

```php
'model' => $listDetails['ip'] ? \BookStack\Actions\Activity::query()->where(['ip' => $listDetails['ip']])->first()
```

Ideally I wouldn't want view-side database queries for information we already have (The searched IP address). I understand that it's akward though due to how the `text.blade.php` file expects a `$model`. 
For a non-database hack I think you could instead do the following:

```php
'model' => (object) $listDetails
```

Reply:
I will use `(object) $listDetails` instead. I was ran out of ideas.

And I think I also need to change the name to `audit_table_ip_address` for translation.